src/components/SQForm/SQFormRadioButtonGroup.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Grid from '@material-ui/core/Grid';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import InputLabel from '@material-ui/core/InputLabel';
+import {SQFormRadioButtonGroupItem} from '../../../src';
+import {useForm} from './useForm';
+
+function SQFormRadioButtonGroup({
+  name,
+  onChange = () => {},
```

Comment:
Let's leave this `undefined` (no default value)
See this line in `useForm`

https://github.com/SelectQuoteLabs/SQForm/blob/755d6e9b2df812daf500f3ce8d7bfc4dab78a947/src/components/SQForm/useForm.js#L54